akka-actor-typed/src/main/scala/akka/actor/typed/internal/ActorContextImpl.scala

```diff
@@ -57,10 +57,10 @@ import scala.concurrent.ExecutionContextExecutor
     internalSpawnAdapter(f, "")
 
   override def spawnAdapter[U](f: java.util.function.Function[U, T]): akka.actor.typed.ActorRef[U] =
-    internalSpawnAdapter(f.apply _, "")
+    internalSpawnAdapter(f.apply, "")
```

Comment:
isn't the `_` to avoid compiler warnings though?

Reply:
Dont think so, there aren't any compiler warnings in the typed module

Reply:
Ok, cool then 👍